model/BookletClassService.php

```diff
@@ -136,22 +137,30 @@ public function getAttachment(KernelResource $booklet): string
      * @param string                       $bookletLabel
      *
      * @return common_report_Report
-     * @throws core_kernel_persistence_Exception
+     * @throws common_exception_Error
      */
     public function updateInstanceAttachment($instance, $tmpFile, string $bookletLabel = '')
     {
-        $report = new common_report_Report(common_report_Report::TYPE_SUCCESS);
+        $report = common_report_Report::createSuccess(
+            __('PDF File for booklet \'%s\' updated', $bookletLabel ?? $instance->getLabel())
```

Comment:
`$bookletLabel` cannot be a `null`. Did you mean to use a ternary operator here?
```suggestion
            __('PDF File for booklet \'%s\' updated', $bookletLabel ?: $instance->getLabel())
```

Reply:
Yes, you are right.